src/main/scala/ai/privado/languageEngine/java/tagger/PrivadoTagger.scala

```diff
@@ -97,9 +96,8 @@ class PrivadoTagger(cpg: Cpg) extends PrivadoBaseTagger {
     if (!ScanProcessor.config.ignoreInternalRules) {
       // Adding custom rule to cache
       StorageInheritRule.rules.foreach(ruleCache.setRuleInfo)
-      StorageInheritRule.rules.foreach(ruleCache.addStorageRuleInfo)
       println(s"${Calendar.getInstance().getTime} - --CustomInheritTagger invoked...")
-      new CustomInheritTagger(cpg, ruleCache).createAndApply()
+      new InheritMethodTagger(cpg, ruleCache).createAndApply()
```

Comment:
You can name this CustomInheritTagger, changing this will fail the comparison script